cli/pkg/kctrl/util/executeCmd.go

```diff
@@ -0,0 +1,35 @@
+package util
```

Comment:
lets move this "util" to builder package.

Reply:
Adopted